package.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "root",
+  "private": true,
+  "workspaces": [
+    "demos/*",
+    "shared/*",
+    "packages/**/*"
+  ],
+  "scripts": {
+    "build": "npx lerna run build",
+    "publish": "npm run build && npx lerna publish --registry http://localhost:4873"
```

Comment:
What is running on http://localhost:4873?

Reply:
Please add a few words to the README about how this monorepo is used as a developer, e.g. about those scripts and the usage of lerna and nx